.github/workflows/gradle-build.yaml

```diff
@@ -0,0 +1,28 @@
+name: Gradle
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Build with Gradle
+        run: ./gradlew --continue verifyGoogleJavaFormat build connectedCheck
+      - if: github.event_name == 'push'
```

Comment:
I suspect you'll need to swap name and if lines, preserving the leading `-`

Reply:
If only there was a way to test these before merging :)

I don't think it should matter, but I was thinking about it for readability anyway since it makes sense to lead with the name. Done. I'm expecting I'll likely need to follow up with another tweak or two once this is merged anyway.